Dropout.lua

```diff
@@ -0,0 +1,70 @@
+local Dropout, Parent = nn.Dropout, nn.Module
```

Comment:
@jnhwkim Okay that is awesome. Can we include that bit of doc in the header of the dropout file? Thanks.